core/meta/src/TClass.cxx

```diff
@@ -1328,6 +1328,8 @@ void TClass::Init(const char *name, Version_t cversion,
 {
    if (!gROOT)
       ::Fatal("TClass::TClass", "ROOT system not initialized");
+   if (!name || !name[0])
```

Comment:
The TClass object also need to be set in a 'non-working' state.  Please use
```
if (!name || !name[0]) {
   ::Error("TClass::Init", "The name parameter is invalid (null or empty)");
   MakeZombie();
   return;
}
``` 
Thanks.

Reply:
OK, will do it that way. Thanks for the feedback